Mlem/Views/Shared/Instance/InstanceStatsView.swift

```diff
@@ -0,0 +1,270 @@
+                    "NSFW Content",
+                    systemImage: Icons.blurNsfw,
+                    value: instance.allowsNSFW ?? false
+                )
+                Divider()
+                settingRow(
+                    "Downvotes",
+                    systemImage: Icons.downvote,
+                    value: instance.allowsDownvotes ?? false
+                )
+                Divider()
+                settingRow(
+                    "Community Creation",
+                    systemImage: "house",
+                    value: instance.allowsCommunityCreation ?? false
+                )
+                Divider()
+                settingRow(
+                    "Slur Filter",
+                    systemImage: Icons.filterFill,
+                    value: instance.slurFilterRegex != nil
+                )
+                if developerMode, let regex = instance.slurFilterString {
+                    Divider()
+                    Text(showingSlurRegex ? regex : "Tap to show slur filter regex")
+                        .textSelection(.enabled)
+                        .font(.footnote)
+                        .foregroundStyle(.secondary)
+                        .padding(12)
+                        .onTapGesture {
+                            showingSlurRegex.toggle()
+                        }
+                }
+                if developerMode, let feedType = instance.defaultFeedType {
+                    Divider()
+                    settingRow(
+                        "Default Feed Type",
+                        systemImage: Icons.feeds,
+                        value: feedType.rawValue
+                    )
+                }
+            }
+            .frame(maxWidth: .infinity)
+            .background(Color(uiColor: .secondarySystemGroupedBackground))
+            .cornerRadius(AppConstants.largeItemCornerRadius)
+            
+            if developerMode {
+                VStack(alignment: .leading, spacing: 0) {
+                    settingRow(
+                        "Show Mod Names in Modlog",
+                        systemImage: Icons.moderation,
+                        value: !(instance.hideModlogModNames ?? true)
+                    )
+                    Divider()
+                    settingRow(
+                        "Applications Email Admins",
+                        systemImage: Icons.person,
+                        value: instance.applicationsEmailAdmins ?? false
+                    )
+                    Divider()
+                    settingRow(
+                        "Reports Email Admins",
+                        systemImage: Icons.moderationReport,
+                        value: instance.reportsEmailAdmins ?? false
+                    )
+                }
+                .frame(maxWidth: .infinity)
+                .background(Color(uiColor: .secondarySystemGroupedBackground))
+                .cornerRadius(AppConstants.largeItemCornerRadius)
+            }
+            
```

Comment:
Currently this view leaves a block of white space at the bottom:

<img width="387" alt="Screenshot 2024-01-27 at 8 18 33 PM" src="https://github.com/mlemgroup/mlem/assets/44140166/babf0a42-44e9-4af2-940b-b106ab2d72ec">

I like having a little bit of overscroll, but imo that space should definitely be gray as well

Reply:
Making that space gray means adding a duo-tone background, which we had issues with in the Community View?

Reply:
Also, I switched from a gray background to a hard cut-off for the Community stats page in #879. I think the styiling on this pages should match, ideally.

Reply:
Yeah, this comment's outdated--reviewed this PR before 879, edited my original review but forgot to delete this comment 🤦